### PR TITLE
DAT-3944 | Infobox Europa styles

### PR DESCRIPTION
**Pull request by dianafa**
## Links

https://wikia-inc.atlassian.net/browse/DAT-3944

## Reviewers

@RafLeszczynski @vforge

### CODE REVIEW

File: front/main/app/styles/module/article/_portable-infobox.scss [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
 $infobox-image-collection-button-width: 60px;
 
 //fonts used in infobox
-$infobox-title-font-size: 18px;
-$infobox-title-line-height: 28px;
+$infobox-title-font-size: 20px;
+$infobox-title-line-height: 24px;
 $infobox-default-font-size: 14px;
 $infobox-default-line-height: 20px;
+$hero-title-font-size: 24px;
```
Comment (RafLeszczynski) [inline front/main/app/styles/module/article/_portable-infobox.scss]: you could use `$type-small`

File: front/main/app/styles/module/article/_portable-infobox.scss [hunk 2]
```diff
@@ -1,20 +1,27 @@
...
+$hero-title-line-height: 28px;
```
Comment (RafLeszczynski) [inline front/main/app/styles/module/article/_portable-infobox.scss]: you could use `$line-height-small` from style guide

File: front/main/app/styles/module/article/_portable-infobox.scss [hunk 3]
```diff
@@ -1,20 +1,27 @@
...
 $min-hero-item-height: 110px;
 
 $infobox-image-collection-button-width: 60px;
 
 //fonts used in infobox
-$infobox-title-font-size: 18px;
-$infobox-title-line-height: 28px;
+$infobox-title-font-size: 20px;
+$infobox-title-line-height: 24px;
 $infobox-default-font-size: 14px;
```
Comment (RafLeszczynski) [inline front/main/app/styles/module/article/_portable-infobox.scss]: you could use `$type-smallest` from style guide

File: front/main/app/styles/module/article/_portable-infobox.scss [hunk 4]
```diff
@@ -1,20 +1,27 @@
...
 $infobox-default-line-height: 20px;
```
Comment (RafLeszczynski) [inline front/main/app/styles/module/article/_portable-infobox.scss]: you could use `$line-height-smallest` from style guide

File: front/main/app/styles/module/article/_portable-infobox.scss [hunk 5]
```diff
@@ -137,6 +145,11 @@ $hero-caption-line-height: 18px;
 			z-index: 1;
 		}
 
+		.pi-hero-title{
```
Comment (RafLeszczynski) [inline front/main/app/styles/module/article/_portable-infobox.scss]: missing space before `{` 
please consider running scss linter on this fie

File: front/main/app/styles/module/article/_portable-infobox.scss [hunk 6]
```diff
@@ -215,10 +233,26 @@ $hero-caption-line-height: 18px;
 				line-height: inherit;
 				list-style-position: inside;
 			}
+
+			ol,
```
Comment (RafLeszczynski) [inline front/main/app/styles/module/article/_portable-infobox.scss]: do we want to apply this style for all lists or only for the direct children?

**Comment by RafLeszczynski** [comment]
do we want to replace the default theme on mercury and use europa for all infoboxes?

File: front/main/app/styles/module/article/_portable-infobox.scss [hunk 7]
```diff
@@ -1,22 +1,29 @@
-$infobox-item-margin: 6px;
-$infobox-background: $color-gray-light;
-$infobox-section-secondary-background: $color-blue-gray-light;
+// 5% button color, 5% black and 90% white
```
Comment (RafLeszczynski) [inline front/main/app/styles/module/article/_portable-infobox.scss]: what do you think about sorting all variables in alphabetical order?

**Comment by RafLeszczynski** [comment]
one question except that LGTM

File: front/main/app/styles/module/article/_portable-infobox.scss (hunk 7 above)
Comment (dianafa) [inline front/main/app/styles/module/article/_portable-infobox.scss]: in my opinion more readable is like now: background colors, colors, hero colors, margin, heights

Comment (vforge) [inline front/main/app/styles/module/article/_portable-infobox.scss]: Can you add comments that will separate those groups you've mentioned?

**Comment by vforge** [comment]
Just a minor comment from me.

**Comment by RafLeszczynski** [comment]
LGTM

**Pull request merged**